src/main/java/uk/gov/hmcts/reform/et/syaapi/models/CaseDocument.java

```diff
@@ -14,9 +15,32 @@
 @Builder
 @Jacksonized
 public class CaseDocument {
+    @JsonProperty("classification")
+    String classification;
+    @JsonProperty("size")
+    String size;
+    @JsonProperty("mimeType")
+    String mimeType;
     @JsonProperty("originalDocumentName")
     String originalDocumentName;
-
+    @JsonProperty("hashToken")
+    String hashToken;
+    @JsonProperty("createdOn")
+    String createdOn;
+    @JsonProperty("createdBy")
+    String createdBy;
+    @JsonProperty("lastModifiedBy")
+    String lastModifiedBy;
+    @JsonProperty("modifiedOn")
+    String modifiedOn;
+    @JsonProperty("ttl")
+    String ttl;
+    @JsonProperty("metadata")
+    Map<String, String> metadata;
     @JsonProperty("_links")
     Map<String, Map<String, String>> links;
+
+    public URI getUri() {
+        return URI.create(links.get("self").get("href"));
+    }
```

Comment:
This should have JavaDoc as until I read the actual code I didn't know it was getting the URI for the links property named "self"